services/workflows-service/src/collection-flow/collection-flow.service.ts

```diff
@@ -229,6 +238,45 @@ export class CollectionFlowService {
     return flowData;
   }
 
+  private async __persistFileUrlsToDocuments(
+    documents: TDocument[] = [],
+    projectIds: TProjectIds,
+  ): Promise<TDocument[]> {
```

Comment:
On `TDocument` `pages` were omitted, let's use `DefaultContextSchema['documents']` until we decide on a single source of truth. With this change we can remove all the types below related to pages.

Reply:
Code is currently tested,type update will force me to update code which eventually could lead to issues.
So lets move forward,this place doesnt looks like issue which affects anything in current scope